docs/recipes/lost-pixel-oss/element-locator-for-components.md

```diff
@@ -0,0 +1,37 @@
+# Element locator for components
+
+In case you build something like a design system, you can use the `elementLocator` property to take a screenshot of the exact component you want to test.
+
+This property is a CSS selector that will be used to find the element in the DOM. The screenshot will be taken of the first element that matches the selector.
+
+You can add this property to `lostpixel.config.ts` like so:
+
+### lostpixel.config.ts - works for all modes
+
+```ts
+// @ts-nocheck
+
+import { CustomProjectConfig } from 'lost-pixel';
+
+export const config: CustomProjectConfig = {
+  storybookShots: {
+    storybookUrl: './storybook-static',
+  },
+  elementLocator: '#root > *:first-child',
```

Comment:
@pedrosousa13 let's start with moving this to the `storybookShots`, as it I envision this feature to be more coupled with the modes rather than overall runs. In case somebody runs DS and wants this in their storybook, they might not need it in their page tests.

Reply:
sure, makes sense